likelihood/kmax_test/kmax_abacus.yml

```diff
@@ -70,65 +19,10 @@ likelihood:
     # Input sacc file
     input_file: choose_input_file
     # List all relevant bins
-    bins:
-      - name: cl1
-      - name: cl2
-      - name: cl3
-      - name: cl4
-      - name: cl5
-      - name: sh1
-      - name: sh2
-      - name: sh3
-      - name: sh4
-      - name: sh5
+    bins: choose_bins
```

Comment:
not sure what's going in this file

Reply:
This is the default config file for the automated runscript `kmax_test.py` described in https://github.com/LSSTDESC/DESCBiasChallenge/wiki/Running-the-minimizer. Basically with this thing you can specify the bin combination you would like to run from the command line.